streams/src/main/java/org/apache/kafka/streams/kstream/Windows.java

```diff
@@ -56,7 +56,7 @@ public String name() {
     }
 
     /**
-     * Set the window emit duration in milliseconds of system time
+     * Set the window emit duration in milliseconds of system time.
```

Comment:
Nitpick:  Is "system time" really enough to define the contract?  Shouldn't we rather say "milliseconds since the epoch"?  We could then add an example/note that informs users that, in Java, you can get the current time in milliseconds since the epoch by calling `System.currentTimeMillis`.